content/applications/sales/point_of_sale/payment/six.rst

```diff
@@ -24,11 +24,11 @@ remove the "Apps" filter and search for "POS Six".
     :align: center
 
 Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`,
-Create a new payment method for SIX, select the payment terminal option
-"SIX without IoT Box", and enter your payment terminal IP address.
+Create a new payment method for SIX, select the appropriate journal, select *SIX* in the **Use a Payment Terminal**
+field, then enter your payment terminal IP address.
```

Comment:
- Create -> create
- Don't start a new line unless you've reached the 100th character.
- Start a new line before you reach the 100th character.

```rst
Back in :menuselection:`Point of Sale --> Configuration --> Payment Methods`, create a new payment
method for SIX, select the appropriate journal, select *SIX* in the **Use a Payment Terminal**
field, then enter your payment terminal IP address.
```